### PR TITLE
feat(anta.tests): Add new AAA test cases

### PR DESCRIPTION
**Pull request by carl-baillargeon**
Hello,

Please review this PR for new AAA test cases.

Thanks

### CODE REVIEW

File: anta/tests/aaa.py [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+    """
+    if len(auth_types) > len(valid_auth_types):
+        raise ValueError(f"Too many parameters provided in auth_types. Valid parameters are: {valid_auth_types}")
+
+    for auth_type in auth_types:
+        if auth_type not in valid_auth_types:
+            raise ValueError(f"Wrong parameter provided in auth_types. Valid parameters are: {valid_auth_types}")
+
+
+# FIXME: AntaTestTemplate does not support 2 templates
```
Comment (titom73) [inline anta/tests/aaa.py:42]: Not sure if we should keep this todo in the code, or open an issue to track as it seems to be a major change required to support this feature.

**Review by titom73 — COMMENTED** [review]
Looks very good. Just wondering if we need to keep comments as it is related to a major change (we could use issue to track it)

File: anta/tests/aaa.py (hunk 1 above)
Comment (carl-baillargeon) [reply anta/tests/aaa.py:42]: Agreed let's open an issue!

**Review by titom73 — APPROVED** [review]
LGTM

**Pull request merged**